src/Sentry.Maui/SentryMauiOptions.cs

```diff
@@ -23,18 +23,14 @@ public SentryMauiOptions()
 #if !PLATFORM_NEUTRAL
         CacheDirectoryPath = Microsoft.Maui.Storage.FileSystem.CacheDirectory;
 #endif
-        AddDefaultEventBinder<MauiButtonEventsBinder>();
-        AddDefaultEventBinder<MauiImageButtonEventsBinder>();
-        AddDefaultEventBinder<MauiGestureRecognizerEventsBinder>();
-        AddDefaultEventBinder<MauiVisualElementEventsBinder>();
```

Comment:
This may seem weird, unless you read https://github.com/getsentry/sentry-dotnet/pull/4294#issuecomment-3094833660